react/src/Tooltip/Tooltip.tsx

```diff
@@ -11,7 +11,7 @@ export interface TooltipProps extends ChakraTooltipProps {
   /**
    * Text which appears on hover/focus.
    */
-  label: string
+  label: React.ReactNode
```

Comment:
We can probably remove this prop if there is no need to be more specific, since the props are already extending chakra’s props

Reply:
Oh, I thought it was added to design system to make the `label` prop required. If that isn't the intention, we can remove it bah haha

Reply:
@karrui can lmk if we wanna make `label` a required prop bah, I'll change accordingly

Reply:
Can remove imho!